### PR TITLE
retry syncWriter once (after 5 seconds)

### PR DESCRIPTION
**Pull request by mrose17**
fixes: https://github.com/brave/browser-laptop/issues/3960

reviewer: @diracdeltas

### CODE REVIEW

File: app/ledger.js [hunk 1]
```diff
@@ -1170,15 +1154,21 @@ var cacheReturnValue = () => {
...
     cb = options
     options = null
   }
   options = underscore.defaults(options || {}, { encoding: 'utf8', mode: parseInt('644', 8) })
 
+  if (syncingP[path]) {
+    if (options.noRetryP) return
+
+    options.noRetryP = true
+    console.log('\nretrying ' + path)
```
Comment (diracdeltas) [inline app/ledger.js]: should this log be suppressed unless there's a debug/verbose flag?

Comment (mrose17) [inline app/ledger.js]: indeed. on the way.

Comment (mrose17) [inline app/ledger.js]: more more accurately, not needed!

File: app/ledger.js [hunk 2]
```diff
@@ -1164,7 +1164,6 @@ var syncWriter = (path, obj, options, cb) => {
     if (options.noRetryP) return
 
     options.noRetryP = true
-    console.log('\nretrying ' + path)
```
Comment (diracdeltas) [inline app/ledger.js]: for follow-up changes in a PR, you can just commit, rebase -i, then force-push, so there is no extra rebase step necessary when review is done

**Comment by diracdeltas** [comment]
please squash commits; otherwise lgtm

**Comment by mrose17** [comment]
superceded by https://github.com/brave/browser-laptop/pull/3966

**Pull request closed without merging**